src/Joystick/Joystick.cc

```diff
@@ -1042,6 +1043,7 @@ void Joystick::_executeButtonAction(const QString& action, bool buttonDown)
         if (buttonDown) emit landingGearRetract();
     } else {
         if (buttonDown && _activeVehicle) {
+            emit unknownAction(action);
```

Comment:
I would leave this as a qCWarning unless you had a specific use case for this signal in mind

Reply:
The signal is intentional. The main point of this PR is to allow custom plugin to register custom joystick actions during runtime and then also to handle them via the emitted signal.